README.md

```
@@ -11,7 +11,7 @@ WebAssembly binding for [llama.cpp](https://github.com/ggerganov/llama.cpp)
 For changelog, please visit [releases page](https://github.com/ngxson/wllama/releases)
 
 > [!IMPORTANT]  
-> **🔥🔥 V3 is out, with multimodal and tool calling support. Read more [here](./guides/intro-v3.md)**  
+> **🔥🔥 V3 is out, with WebGPU, multimodal and tool calling support. Read more [here](./guides/intro-v3.md)**  
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)